### PR TITLE
GH-884 Adds surfaceBorderWidth to Grid

### PR DESCRIPTION
**Pull request by jbrundage**
Fixes GH-884

@mlzummo @GordonSmith Please review

https://rawgit.com/jbrundage/Visualization/GH-884-GridCellBorders/demos/dermatology.html?src/layout/Grid&surfaceBorderWidth=4

Signed-off-by: Jay Brundage <jaman.brundage@lexisnexis.com>

### CODE REVIEW

**Comment by mlzummo** [comment]
looks good

File: src/config.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ require.config({
         "dagre": "../bower_components/dagre/index",
         "topojson": "../bower_components/topojson/topojson",
         "colorbrewer": "../bower_components/colorbrewer/colorbrewer",
-        "d3-cloud": "../bower_components/d3-cloud/d3.layout.cloud",
+        "d3-cloud": "../bower_components/d3-cloud/build/d3.layout.cloud",
```
Comment (GordonSmith) [inline src/config.js]: Not needed with my new PR

File: src/config.js [hunk 2]
```diff
@@ -98,7 +98,7 @@ case "cdn.rawgit.com":
             "dagre": "//" + window.location.hostname + "/cpettitt/dagre/v0.7.1/dist/dagre.min",
             "topojson": "//" + window.location.hostname + "/mbostock/topojson/v1.6.19/topojson",
             "colorbrewer": "//" + window.location.hostname + "/jeanlauliac/colorbrewer/v1.0.0/colorbrewer",
-            "d3-cloud": "//" + window.location.hostname + "/jasondavies/d3-cloud/v1.1.2/d3.layout.cloud",
+            "d3-cloud": "//" + window.location.hostname + "/jasondavies/d3-cloud/v1.2.0/build/d3.layout.cloud",
```
Comment (GordonSmith) [inline src/config.js]: Not needed.

File: src/other/WordCloud.js [hunk 3]
```diff
@@ -5,7 +5,7 @@
     } else {
         root.other_WordCloud = factory(root.d3, root.common_SVGWidget, root.other_IWordCloud);
     }
-}(this, function (d3, SVGWidget, IWordCloud) {
+}(this, function (d3, SVGWidget, IWordCloud, d3Cloud) {
```
Comment (GordonSmith) [inline src/other/WordCloud.js]: Not needed

File: src/other/WordCloud.js [hunk 4]
```diff
@@ -38,7 +38,12 @@
     };
 
     WordCloud.prototype.enter = function (domNode, element) {
-        this.cloud = d3.layout.cloud()
+        if (typeof define === "function" && define.amd) {
+            this.cloud = d3Cloud();
+        } else {
+            this.cloud = d3.layout.cloud();
+        }
+        this.cloud
```
Comment (GordonSmith) [inline src/other/WordCloud.js]: Not needed

File: test/runner_build.html [hunk 5]
```diff
@@ -20,7 +20,8 @@
             paths: {
                 "src": "../dist-amd",
                 "test": "../test",
-                "font-awesome": "../dist-amd/font-awesome/css/font-awesome.min"
+                "font-awesome": "../dist-amd/font-awesome/css/font-awesome.min",
+                "amchartsImg": "../dist-amd/img/amcharts/"
```
Comment (GordonSmith) [inline test/runner_build.html]: Should be part of diff PR

File: test/runner_build_nonamd.html [hunk 6]
```diff
@@ -28,7 +28,7 @@
     <script src="../bower_components/dagre/index.js"></script>
     <script src="../bower_components/colorbrewer/colorbrewer.js"></script>
     <script src="../bower_components/topojson/topojson.js"></script>
-    <script src="../bower_components/d3-cloud/d3.layout.cloud.js"></script>
+    <script src="../bower_components/d3-cloud/build/d3.layout.cloud.js"></script>
```
Comment (GordonSmith) [inline test/runner_build_nonamd.html]: Not needed.

**Pull request merged**